frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseSlideOutBranchAction.java

```diff
@@ -20,6 +29,8 @@
       IExpectsKeyGitMacheteRepository,
       IExpectsKeyProject {
 
+  private static final String DELETE_LOCAL_BRANCH_ON_SLIDE_OUT_GIT_CONFIG_KEY = "machete.slideout.deletelocalbranch";
```

Comment:
Nit: git config keys are case-INsensitive, but for clarity:
```suggestion
  private static final String DELETE_LOCAL_BRANCH_ON_SLIDE_OUT_GIT_CONFIG_KEY = "machete.slideOut.deleteLocalBranch";
```